GDAS-validation/run_job.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+# run_job.sh
+# using rocotoboot, point to a global-workflow experiment
+# and run a specified job for a specified cycle
+
+usage() {
+  set +x
+  echo
+  echo "Usage: $0 -c /path/to/config.sh -t taskname"
+  echo
+  echo "  -c  path to config shell script to source"
```

Comment:
`-c` option is more than just the path to the config file to source.  Line 43 sources the `-c` option.  Thus, the `-c` option requires the path and file to source.  If a user specifies the path without the file to actually source, execution of `run_job.sh` generates an error.  For example,

```
Orion-login-2:/work2/noaa/da/rtreadon/gdas-validation/expdir/gdas_eval_satwind_GSI$ /work2/noaa/da/rtreadon/git/jedi-t2o/pr89/GDAS-validation/run_job.sh -c /work2/noaa/da/rtreadon/gdas-validation/expdir/gdas_eval_satwind_GSI -t gdasprep
/work2/noaa/da/rtreadon/git/jedi-t2o/pr89/GDAS-validation/run_job.sh: line 43: source: /work2/noaa/da/rtreadon/gdas-validation/expdir/gdas_eval_satwind_GSI: is a directory
10/13/23 14:15:31 UTC :: .xml :: ERROR: You do not have permission to create //.db
```